Tasks/NuGetInstaller/nugetinstaller.ts

```diff
@@ -73,7 +73,7 @@ async function main(): Promise<void> {
         // locateNuGetExe() will strip them and check for existence there.
         let nuGetPath = tl.getPathInput("nuGetPath", false, false);
         let userNuGetProvided = false;
-        if(tl.filePathSupplied("nuGetPath")){
+        if(nuGetPath !== null && tl.filePathSupplied("nuGetPath")){
```

Comment:
the v1 agent uses a different working directory.  when the path is null, tl.filePathSupplied does not do the proper thing.  We were told we could move from string to filepath as the type in the json but don't want to make that change now.  We've also asked to make filePathSupplied or getPathInput throw on null, if they are only meant to handle file paths (since the agent always roots those, it should never be null).  we were going to bring that up later as a 107 discussion.